types/alexa-sdk/index.d.ts

```diff
@@ -642,80 +785,80 @@ export namespace services {
          * @param itemStatus itemsStatus can be either "active" or "completed"
          * @param token bearer token for list management permission
-         * @returns Promise<object>
+         * @returns Promise<ListObject>
          */
-        getList(listId: string, itemStatus: ListItemObjectStatus, token: string): Promise<object>;
+        getList(listId: string, itemStatus: ListItemObjectStatus, token: string): Promise<ListObject>;
 
         /**
          * Update a custom list. Only the list name or state can be updated
          * @param listId unique Id associated with the list
-         * @param listObject listObject
+         * @param params params
          * @param token bearer token for list management permission
          * @returns Promise<object>
          */
-        updateList(listId: string, listObject: ListObject, token: string): Promise<object>;
+        updateList(listId: string, params: UpdateListParams, token: string): Promise<ListObject>;
 
         /**
          * Delete a custom list
          * @param listId unique Id associated with the list
          * @param token bearer token for list management permission
-         * @returns Promise<object>
+         * @returns Promise<void>
          */
-        deleteList(listId: string, token: string): Promise<object>;
+        deleteList(listId: string, token: string): Promise<void>;
 
         /**
          * Create an item in an active list or in a default list
          * @param listId unique Id associated with the list
-         * @param listItemObject listItemObject
+         * @param params params
          * @param token bearer token for list management permission
-         * @returns Promise<object>
+         * @returns Promise<ListItemObject>
          */
-        createListItem(listId: string, listItemObject: ListItemObject, token: string): Promise<object>;
+        createListItem(listId: string, params: CreateListItemParams, token: string): Promise<ListItemObject>;
 
         /**
          * Retrieve single item within any list by listId and itemId
          * @param listId unique Id associated with the list
          * @param itemId unique Id associated with the item
          * @param token bearer token for list management permission
-         * @returns Promise<object>
+         * @returns Promise<ListItemObject>
          */
-        getListItem(listId: string, itemId: string, token: string): Promise<object>;
+        getListItem(listId: string, itemId: string, token: string): Promise<ListItemObject>;
 
         /**
          * Update an item value or item status
          * @param listId unique Id associated with the list
          * @param itemId unique Id associated with the item
-         * @param listItemObject listItemObject
+         * @param params params
          * @param token bearer token for list management permission
-         * @returns Promise<object>
+         * @returns Promise<ListItemObject>
```

Comment:
Might be easier to just remove type annotations from JSDoc, since we have them in the code anyway.

Reply:
Sounds good. I will get that in in the next batch of updates which is on the way. Thanks